bodhi/client/bindings.py

```diff
@@ -172,6 +174,12 @@ def __init__(self, base_url=BASE_URL, username=None, password=None, staging=Fals
         if base_url[-1] != '/':
             base_url = base_url + '/'
 
+        if debug:
+            ch = logging.StreamHandler()
+            ch.setLevel(logging.DEBUG)
+            log.addHandler(ch)
+            log.setLevel(logging.DEBUG)
```

Comment:
libraries should not set up logging handlers or set log levels. This library is not just used by the bodhi cli and so it should be unopinionated about logging. It's OK for the lib to log things at debug level, but the logger should not be configured by it.

Reply:
Any preference where we could configure it?
I thought this was a good place since it was common with all the actions performed by the CLI, but I'm fine w/ changing this :)